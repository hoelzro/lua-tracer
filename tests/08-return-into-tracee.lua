local tracee

local function deepest()
  tracee(false)
end

--[[local]] function tracee(go_deeper)
  if go_deeper then
    deepest()
  end

  local b = 1 -- trace: go_deeper
end

local function deep()
  tracee(true)
end

deep()
