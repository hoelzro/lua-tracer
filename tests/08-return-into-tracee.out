go_deeper	false
go_deeper	true
